source/apiVolontaria/apiVolontaria/admin.py

```diff
@@ -22,6 +22,7 @@ class UserAdmin(admin.ModelAdmin):
 
 
 admin.site.register(models.TemporaryToken)
+admin.site.register(models.ActivationToken)
```

Comment:
Activation? Isn't this misguiding? I would expect an `activation_token` to be used for account "activation". 

Why not use a subclass of temporary token?

Reply:
`ActivationToken` already exist from the feature of user's account activation by email and token.

I wanted to use the same model for ALL token of action by adding a `type` attribute to avoid multiplication of DB table just for one kind of token.

I think it will be great to rename this `ActivationToken` to `ActionToken` but it's a big refactor so i wanted to see with you before.

Reply:
Temporary token is not relates to this PR.